src/features/Dashboard/components/RecentTransactionList.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { fAccounts } from '@fixtures';
+
+import RecentTransactionList from './RecentTransactionList';
+
+const defaultProps: React.ComponentProps<typeof RecentTransactionList> = {
+  accountsList: fAccounts
+};
+
+export default { title: 'Organisms/RecentTransactionList', component: RecentTransactionList };
+
+export const Default = () => {
```

Comment:
Can we add a state with transactions in it?

Reply:
Currently the component filters account transactions by txs contained in `txHistory`
At the moment `txHistory` is defined in `StoreProvider` so displaying them would require mocking `StoreProvider` for the story.
I prefer to wait that we move `txHistory` to sagas to avoid adding complexity.